src/server/controllers/items.controller.js

```diff
@@ -0,0 +1,28 @@
+const Item = require('../models/item.model');
+
+// Get all the items
+const getAllItems = function(req, resp) {
+  Item.find({}, (err, items) =>
+    err ? resp.status(400).send(err.message) : resp.status(200).json(items)
+  );
+};
+
+// Get all ordered items
+const getOrderedItems = function(req, resp) {
```

Comment:
ordering/filtering should be parameter in getAllItems and both getOrderedItems and getAllUnordered should be removed